src/app/js/formats/hierarchy/Hierarchy.js

```diff
@@ -300,7 +284,12 @@ class Hierarchy extends PureComponent {
                         d.parent.y
                     );
                 })
-                .attr('stroke', color);
+                .attr(
+                    'stroke',
+                    /(^#[0-9A-F]{6}$)|(^#[0-9A-F]{3}$)/i.test(color)
```

Comment:
This limits the color to hexadecimal codes.
Maybe we should create a function `isValidColor` and use it as is now.
That way, we could improve the function later.
And the function could be common to all color parameters.